test/setup_environment.sh

```diff
@@ -449,5 +449,5 @@ if [[ -n ${build_bitcoind} ]]; then
 
     # Do the build
     cmake -B build --toolchain depends/x86_64-pc-linux-gnu/toolchain.cmake -DBUILD_TESTS=OFF -DBUILD_BENCH=OFF
-    cmake --build build --target bitcoind
+    cmake --build build -j $(nproc) --target bitcoind
```

Comment:
Same for building depends a few line above?

Reply:
Done